dali/python/nvidia/dali/_autograph/impl/api.py

```diff
@@ -283,6 +289,20 @@ def is_autograph_artifact(entity):
     return hasattr(entity, "autograph_info__")
 
 
+def is_frame_converted_call(frame_info):
```

Comment:
```suggestion
def is_frame_call_converted(frame_info):
```
?
This order of words is used below.... or is this name supposed to mean something else?